Library/Formula/multimarkdown.rb

```diff
@@ -30,9 +23,15 @@ def install
       system "make"
       bin.install "multimarkdown"
     end
+
+    bin.install Dir["scripts/*"].reject { |f| f =~ /\.bat$/ }
   end
 
   test do
+    Dir["scripts/*"].reject { |f| f =~ /\.bat$/ }.each do |unix_script|
```

Comment:
14 overall, with 8 unix and 6 windows wrappers. I think, when we test the plain "mmd"-wrapper, we should be safe, as the XHTML-output can be treated as text and scrutinized by the test. All other wrappers (PDF, tex, etc.) should work.